modules/openy_gc_personal_training/openy_gc_personal_training.install

```
@@ -122,3 +122,29 @@ function openy_gc_personal_training_update_8004(&$sandbox) {
     }
   }
 }
+
+/**
+ * Add "State" field to "Personal Training" entity.
+ */
+function openy_gc_personal_training_update_8005($sandbox) {
+  $state_field_storage_definition = BaseFieldDefinition::create('state')
```

Let's throw an exception in case the `state_machine` module does not exist